docs/user-guide/cli-swreqplugins.md

```diff
@@ -7,10 +7,11 @@ Before you use Zowe&trade; CLI plug-ins, complete the following steps:
-| [IBM Db2 Database Plug-in for Zowe CLI](cli-db2plugin.md) | <ul> <li>Download and prepare the ODBC driver (required for only package installations) and address the licensing requirements. </li><li>**(MacOS)** Download and Install [Xcode](https://developer.apple.com/xcode/resources/).</li> </ul>|  [z/OS FTP Plug-in for Zowe CLI](cli-ftpplugin.md) | <ul> <li>Ensure that z/OS FTP service is enabled and configured with `JESINTERFACELEVEL` = 2.</li> <li>FTP over SSL is recommended.</li> </ul> |
+| [IBM Db2 Database Plug-in for Zowe CLI](cli-db2plugin.md) | <ul> <li>Download and prepare the ODBC driver (required for only package installations) and address the licensing requirements. _Perform this task before you install the plug-in_.</li><li>**(MacOS)** Download and Install [Xcode](https://developer.apple.com/xcode/resources/).</li> </ul>|  [z/OS FTP Plug-in for Zowe CLI](cli-ftpplugin.md) | <ul> <li>Ensure that z/OS FTP service is enabled and configured with `JESINTERFACELEVEL` = 2.</li> <li>FTP over SSL is recommended.</li> </ul> |
 | [IBM z/OS FTP Plug-in for Zowe CLI](cli-ftpplugin.md) | <ul> <li>Ensure that z/OS FTP service is enabled and configured with `JESINTERFACELEVEL` = 2.</li> <li>FTP over SSL is recommended.</li> </ul> |
 | [IBM IMS Plug-in for Zowe CLI](cli-imsplugin.md) | <ul><li>Ensure that [IBM® IMS™ v14.1.0](https://www.ibm.com/support/knowledgecenter/en/SSEPH2_14.1.0/com.ibm.ims14.doc/ims_product_landing_v14.html) or later is installed and running in your mainframe environment.</li><li> Configure [IBM® IMS™ Connect](https://www.ibm.com/support/knowledgecenter/en/SSEPH2_13.1.0/com.ibm.ims13.doc.ccg/ims_ct_intro.html).</li> <li>Configure [IBM IMS Operations APIs](https://github.com/zowe/ims-operations-api) to enable communication between the CLI and the IMS instance. </li></ul> |
 | [IBM MQ Plug-in for Zowe CLI](cli-mqplugin.md) | <ul><li>Ensure that [IBM® MQ™ v9.1.0](https://www.ibm.com/support/knowledgecenter/en/SSFKSJ_9.1.0/com.ibm.mq.pro.doc/q121910_.htm) or later is installed and running in your mainframe environment. Please read this blog for more information: [Exposing the MQ REST API via the Zowe API Mediation Layer](https://developer.ibm.com/messaging/2019/05/17/exposing-the-mq-rest-api-via-the-zowe-api-mediation-layer/) </li></ul>|
+| [Visual Studio Code Extension for Zowe](../user-guide/ze-install.md#software-requirements) | <ul><li>Ensure that you install [Node.js v8.0](https://nodejs.org/en/download/) or later.</li></ul> |
```

Comment:
We need Node.js 12 or later. We probably want to document Node.js 14 or later, because Node 12 goes out of LTS at the end of April 2022.

Reply:
I believe the Node 8.0 requirement is for Zowe Explorer. I'm not sure whether it should be in this list, since it's a VS Code extension and not a CLI plug-in?